test/mxarray.jl

```diff
@@ -124,10 +124,10 @@ a_mx = mxarray(3.25)
 @test jscalar(a_mx) == 3.25
 delete(a_mx)
 
-a_mx = mxarray(@compat Int32(12))
-@test eltype(a_mx) == Int32
+a_mx = mxarray(12)
+@test eltype(a_mx) == Int
 @test size(a_mx) == (1, 1)
-@test jscalar(a_mx) == @compat Int32(12)
+@test jscalar(a_mx) == Int(12)
```

Comment:
Both 32-bit and 64-bit MATLAB have both `int32` and `int64`. If either of them are broken that would be bad.

There is no `int` type in MATLAB and there are no integer literals, but I would expect that `int64` really is 64 bits on 64-bit MATLAB.